packages/editor-ui/src/components/ExecutionsList.vue

```diff
@@ -229,7 +224,7 @@
 			</div>
 			<div v-else :class="$style.loadedAll">{{ $locale.baseText('executionsList.loadedAll') }}</div>
 		</div>
-		<div v-if="checkAll === true || isIndeterminate === true" :class="$style.selectionOptions">
+		<div v-if="numSelected > 0" :class="$style.selectionOptions">
 			<span>
 				{{ $locale.baseText('executionsList.selected', { interpolate: { numSelected } }) }}
```

Comment:
does not handle plural case. should be `1 execution selected` or `10 executions selected`
<img width="358" alt="Bildschirmfoto 2023-03-06 um 09 50 23" src="https://user-images.githubusercontent.com/4711238/223061708-5d00ffab-0c88-45ed-bc9d-b80b7c25b20f.png">

Reply:
Done by Csaba